gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/AuthConfig.groovy

```diff
@@ -41,11 +40,19 @@ class AuthConfig {
   }
 
   static void configure(HttpSecurity http) throws Exception {
-    http.csrf().disable()
-    http.authorizeRequests()
+    http
+      .authorizeRequests()
         .antMatchers(HttpMethod.OPTIONS, "/**").permitAll()
         .antMatchers('/auth/user').permitAll()
         .antMatchers('/health').permitAll()
         .antMatchers('/**').authenticated()
+        .and()
+      .logout()
+        .logoutUrl("/auth/logout")
```

Comment:
There is no `/auth/logout` endpoint in the controller. Is this what Spring expects to use as the logout URI?